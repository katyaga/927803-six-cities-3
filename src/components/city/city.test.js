import React from "react";
import renderer from "react-test-renderer";
import City from "./city.jsx";

const MockData = {
  city: `Paris`,
  activeCity: `Brussels`,
};

it(`Should Review render correctly`, () => {
  const tree = renderer
    .create(<City
      key={2}
      activeCity={MockData.activeCity}
      city={MockData.city}
      handleCityClick={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
