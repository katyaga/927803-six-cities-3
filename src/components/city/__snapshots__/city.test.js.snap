// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Review render correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item "
    onClick={[Function]}
  >
    <span>
      Paris
    </span>
  </a>
</li>
`;
